conformance_tests/tools/debug/src/test_debug_utils.cpp

```diff
@@ -831,6 +832,7 @@ void zetDebugMemAccessTest::run_read_write_module_and_memory_test(
       buffer[0] = 0;
       lzt::debug_write_memory(debugSession, thread, memorySpaceDesc, sizeToRead,
                               buffer);
+      lzt::clear_exceptions(device, debugSession, thread);  
```

Comment:
for example here, `thread` is an all thread. So either iterate in `clear_exceptions` or change to `stopped_thread`